modes/config-mode/public/theme/theme.json5

```diff
@@ -4,14 +4,26 @@
 
   ConfigurableModes: {
     modes: {
-      "@ohif/longitudinal": "@ohif/mode-longitudinal",
+      "hpMode": "hpMode",
```

Comment:
Just for curiosity, what is the difference between defining as a string value or an object as such "ecgMode"?

Reply:
The string one does a runtime lookup of the value in config point - the problem is they may still be changing until actually retrieved, so using a string lookup allows for hte changes.  This area is still in flux quite a bit, but will have to document it more once it stabilizes.